voxelbotutils/cogs/owner_only.py

```diff
@@ -152,6 +152,38 @@ async def reload(self, ctx:utils.Context, *cog_name:str):
             await ctx.send('```py\n' + traceback.format_exc() + '```')
             return
         await ctx.send('Cog reloaded.')
+        
+    @commands.command(aliases=['dlcog'], cls=utils.Command)
+    @commands.is_owner()
+    async def downloadcog(self, ctx: utils.Context, url: str):
```

Comment:
in my opinion i dont see the need for a space between `ctx: and utils.Context`, it doesn't align with the rest of the file.

Reply:
so like `async def downloadcog(self, ctx:utils.Context, url:str):`